### DISCUSSION TITLE
Twice all the tree structure disappeared for a window, is this a feature I accidentally enable, or some bug?

### DISCUSSION THREAD

**Discussion by mzso** [piroor/treestyletab#3688] [Q&A]
I had this annoying event twice. All of the sudden I see all the tabs going on the top level.
It wasn't just a display issue. I could only restore the tree structure by restoring the last session from the backups.

Not sure what happens, but I don't see a hotkey for TST that does this.

**Comment by irvinm** [comment]
It does happen for some, but certainly shouldn't be frequent.  ("Tabs become flatten suddenly / tree structure is lost accidentally" in the [FAQ](https://github.com/piroor/treestyletab/blob/trunk/README.md#troubles-unexpected-behaviors))
